src/ui/camera.js

```diff
@@ -90,8 +90,7 @@ export type CameraOptions = {
     bearing?: number,
     pitch?: number,
     around?: LngLatLike,
-    padding?: PaddingOptions,
-    offset?: PointLike
```

Comment:
Is this deletion in passing of an unused property?

Reply:
Yeah, that was accidentally added in #11524. We don't support `offset` in `CameraOptions`, so there is no way to use `offset` with `jumpTo`. One related thing I've stumbled on is #7902, but we can address it in a separate PR.